geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/index/AttributeIndex.scala

```diff
@@ -269,6 +269,9 @@ object AttributeIndex {
   // store 2 bytes for the index of the attribute in the sft - this allows up to 32k attributes in the sft.
   def indexToBytes(i: Int): Array[Byte] = Array((i << 8).asInstanceOf[Byte], i.asInstanceOf[Byte])
 
+  // TODO verify this
+  def bytesToIndex(b0: Byte, b1: Byte): Short = ((b0 >> 8) + b1).toShort
```

Comment:
may need to mask of negative sign of b1 if its there? hmmm ```b1 & 0xFF``` but yeah this might need a little unit test and verification.